bin/fpm-readiness.sh

```diff
@@ -1,13 +1,53 @@
 #!/usr/bin/env ash
 
-# Send a ping request via fast-cgi. Use an empty environment with `env -i`.
-PING_RESPONSE=$(env -i SCRIPT_NAME=/ping SCRIPT_FILENAME=/ping REQUEST_METHOD=GET cgi-fcgi -bind -connect /sock/fpm.sock | tail -1);
+# 1️⃣ Send a ping request via fast-cgi. Use an empty environment with `env -i`.
+PING_RESPONSE=$(env -i SCRIPT_NAME=/ping SCRIPT_FILENAME=/ping REQUEST_METHOD=GET cgi-fcgi -bind -connect /sock/fpm.sock &);
```

Comment:
Is this the Async issue I was referring to earlier? 🤔

Reply:
I'm not sure... It's been a long day 🙃 I've added some notes to the Miro board.

We did talk about the potential for this script to be bringing down the container.

I tried to replicate a failure on staging and ran this script 10k times in 2 mins and it did not cause a CPU spike of the shape that we are trying to debug.